webpack.config.dev.js

```diff
@@ -31,7 +31,8 @@ export default {
         removeComments: true,
         collapseWhitespace: true
       },
-      inject: true
+      // injection is now handled in the template to allow for absolute paths, fixes issue with nested routes
```

Comment:
I'm concerned that setting inject to false disables **all** injection, which means we have to provide manual injections for all potential file types. For example, would Bootstrap related filetypes need to be explicitly injected in the .ejs file with this setup?